styles/admin.css

```
@@ -4,33 +4,33 @@
+    --ring: var(--mundia-navy);
+
+    background: var(--mundia-paper);
     @apply text-slate-900;
   }
 }
 
 @layer components {
   .admin-container {
     @apply flex min-w-0 flex-1 flex-col p-4 sm:p-6 lg:px-10 lg:py-8 xl:px-12 2xl:px-16;
-    background: oklch(97% 0.008 218);
+    background: var(--mundia-paper);
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)